plugins/base/src/main/kotlin/translators/descriptors/DefaultDescriptorToDocumentableTranslator.kt

```diff
@@ -219,7 +219,7 @@ private class DokkaDescriptorVisitor(
             extra = PropertyContainer.withAll(
                 descriptor.additionalExtras(),
                 descriptor.getAnnotations(),
-                ConstructorValues(descriptor.getAppliedConstructorParameters())
+                ConstructorValues(mapOf(sourceSet to descriptor.getAppliedConstructorParameters()))
```

Comment:
Let's use the predefined util function to keep consistency `private fun <T> T.toSourceSetDependent()` defined above in the class body.

Source:
https://github.com/Kotlin/dokka/blob/dev-0.11.0/plugins/base/src/main/kotlin/translators/descriptors/DefaultDescriptorToDocumentableTranslator.kt#L102

Reply:
Thanks, fixed